packages/picasso/src/Badge/story/Sizes.example.tsx

```diff
@@ -11,26 +11,26 @@ const Example = () => (
     <Grid>
       <Grid.Item small={6}>
         <Container top='small' bottom='medium' flex>
```

Comment:
```suggestion
        <Container top='small' bottom='medium' flex gap="small">
```

Since container accepts `gap`, we can simplify the examples. We could remove all the inner containers.